tests/fsharp/tests.fs

```diff
@@ -75,7 +69,7 @@ module CoreTests =
 #endif
 
 
-    [<Fact>]
+    [<Fact(Skip="Need to deal with hardcoded paths?")>]
```

Comment:
But so this just disables the whole suite basically? :) I don't know how useful they are, but should we probably keep the old mechanism for them, as in, copy it to here so that it's not used elsewhere?